coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
```

Comment:
That would not work since we can't define the same flag twice: we can't do 
```
parser = argparse.ArgumentParser()
parser.add_argument('-e', help='e flag')
g = parser.add_mutually_exclusive_group()
h = parser.add_argument_group()
g.add_argument('-a', help='a flag')
g.add_argument('-b', help='b flag')
g = parser.add_mutually_exclusive_group()
g.add_argument('-a', help='a flag')
g.add_argument('-c', help='c flag')
```
to achieve `[-e] [-a | -b] [-a | -c]`. It really seems they haven't added this feature yet (see [here](http://bugs.python.org/issue10984) and [here](http://bugs.python.org/issue11588)). So I think we have no choice to go with our own custom conflict checker for now.